src/plugins/intel_gpu/src/kernel_selector/cl_kernels/softmax_gpu_bf.cl

```diff
@@ -4,43 +4,60 @@
 
 #include "include/batch_headers/common.cl"
 
+#if IS_DYNAMIC
+#define CALC_POWER(n) ({uint pos = 0; uint i = n; do { i >>= 1; ++pos; } while (i); --pos;})
```

Comment:
Please undefine this macro at the end of file.

Reply:
add `#undef` directive. thanks!